### DISCUSSION TITLE
Dead Galaxy

### DISCUSSION THREAD

**Discussion by SultanRTG** [endless-sky/endless-sky#9350] [Lore]
Why are some of the most advanced civilizations (technologically) dying or dead already? The Builders and the Rulei were pretty advanced civilizations, but both of them went extinct. Similarly, the Korath are rapidly declining, and the Remnant are suffering, if not being in danger of extinction. The only main correlation between the other tier 2 species is that they have complete control over their factions (Bunrodea have a sole queen as a leader, the Heliarch have 3 consuls that maintain control). The only faction exempt from this are the Wanderers, who have the Pug protecting them (pretty much holding their hand to avoid their extinction).

**Comment by Zitchas** [comment]
Worth noting that the Remnant buck the trend by being one of the few in a state of active evolution. They aren't particularly suffering right now.

**Reply by SultanRTG** [reply]
In one of the missions dialogues (after scanning the Tubfalet prototype i think) the prefect you are talking too seems to think that the Korath are being more aggressive (and alludes to a possible invasion too i believe).

**Reply by pepoluan** [reply]
Yes the Korath are aggressive but the Remnants can hold their own. They never really "in need of your help" w.r.t. the Korath, but they are "very busy" so they are willing to give you some jobs to clear their backlog.

Even in that one mission where you need to evac a recon outpost, it's not because they are unable to, but because all their resources are tied up while this particular evac is time-sensitive. Had they more time they will probably just do it themselves.

**Comment by Saugia** [comment]
Typically it's because of their own hubris. 

The Builders in-lore were destroyed by their own creations, the Ka'het. 

The Rulei I have yet to reveal, but there is a lore reason why they aren't around anymore. 

The Korath fell in their own civil wars, and because they started blowing up stars (also not good for your health), the Drak gave them ultimatums.

The Remnant aren't suffering, they're working from a hard position, they aren't in fallen state, rather, they're rising from a developing one. 

The Wanderers may be protected by the Pug, but there's nothing saying they'd go extinct without them.

The Heliarchs and Bunrodea are in good spots for themselves in terms of their civilizations, but that could change based on future missions as well.

TLDR: Because there are given lore reasons by the relative creators or developers.

**Comment by Arachi-Lover** [comment]
The Heliarchs have a couple hundred consuls, not just 3, it's simply that there is an equal number of consuls of each species, meaning whatever the number is, it's a multiple of 3 as there are 3 species represented.
